https://github.com/kuthulux/gnome-connection-manager/issues/4#issuecomment-614042684

The minimial width is an issue with gtk3. In gtk2 I set a minimal
width/height but then when moving the handle the width/height of
the terminal can be modified, but that doesn't happen now.
To overcome this, apply this patch to avoid setting a minimal size,
instead just set the position of the handle to the middle of both
terminals after splitting them.

index c17b93b..b46ccee 100755
--- a/gnome_connection_manager.py
+++ b/gnome_connection_manager.py
@@ -1651,12 +1651,11 @@ class Wmain(SimpleGladeApp):
             cp  = cnb.get_parent()
 
             if direction==HSPLIT:
-                w,h = cnb.get_allocated_width()/2, cnb.get_allocated_height()
+                p = cnb.get_allocated_width()/2
             else:                
-                w,h = cnb.get_allocated_width(), cnb.get_allocated_height()/2
-
-            cnb.set_size_request(w,h)
-            hp.set_size_request(nb.get_allocated_width(), cnb.get_allocated_height())
+                p = cnb.get_allocated_height()/2
+            hp.set_position(p)
+            hp.set_wide_handle(True)
             
             cp.remove(cnb)
             cp.add(hp)

